### PR TITLE
Moreschemachanges

### PR DESCRIPTION
**Pull request by bendk**
Several more schema changes to items.

I'd love some peer review on this one, but please don't merge until janet also tests it out.

### CODE REVIEW

**Comment by bendk** [comment]
Janet has tested most of this (minus the last 4 commits or so).  I think that's good enough to merge if it passes peer-review.

File: tv/lib/databaseupgrade.py [hunk 1]
```diff
@@ -3828,3 +3828,124 @@ def get_title(metadata_title, torrent_title, entry_title, filename):
         if title != metadata_title:
             cursor.execute("UPDATE item SET title=? WHERE id=?", (title, id_))
 
+def upgrade180(cursor):
+    # Rename columns in the item table
+    rename_columns = {
+        'autoDownloaded': 'auto_downloaded',
+        'pendingManualDL': 'pending_manual_dl',
```
Comment (paulswartz) [inline tv/lib/databaseupgrade.py]: If we're going to rename it anyways, why not `pending_manual_download`?

File: tv/lib/databaseupgrade.py [hunk 2]
```diff
@@ -3828,3 +3828,124 @@ def get_title(metadata_title, torrent_title, entry_title, filename):
...
+    cursor.execute("ALTER TABLE remote_downloader "
+                   "ADD COLUMN short_filename text")
+    cursor.execute("ALTER TABLE remote_downloader "
+                   "ADD COLUMN filename text")
+    cursor.execute("ALTER TABLE remote_downloader "
+                   "ADD COLUMN reason_failed text")
+    cursor.execute("ALTER TABLE remote_downloader "
+                   "ADD COLUMN short_reason_failed text")
+    cursor.execute("ALTER TABLE remote_downloader "
+                   "ADD COLUMN dler_type text")
```
Comment (paulswartz) [inline tv/lib/databaseupgrade.py]: `downloader_type`?

**Comment by bendk** [comment]
Added some commits based on the review.  If it looks good, I think we can merge.  Janet's tested it.

**Pull request merged**